manuscripts2/report.py

```diff
@@ -55,23 +64,23 @@ class Report():
 
     # Helper dict to map a data source class with its Elasticsearch index
     class2index = {
-        git.GitMetrics: GIT_INDEX,
-        github_issues.IssuesMetrics: GITHUB_ISSUES_INDEX,
-        github_prs.PullRequestsMetrics: GITHUB_PRS_INDEX,
+        git: GIT_INDEX,
+        github_issues: GITHUB_ISSUES_INDEX,
+        github_prs: GITHUB_PRS_INDEX
     }
 
     # Helper dict to map a data source name with its python class
     ds2class = {
-        "git": git.GitMetrics,
-        "github_issues": github_issues.IssuesMetrics,
-        "github_prs": github_prs.PullRequestsMetrics,
+        "git": git,
+        "github_issues": github_issues,
+        "github_prs": github_prs
     }
 
     def __init__(self, es_url=None, start=None, end=None, data_dir=None, filters=None,
                  interval="month", offset=None, data_sources=None,
                  report_name=None, projects=False, indices=[], logo=None):
 
-        Query.interval_ = interval
+        self.interval = Query.interval_ = interval
```

Comment:
In general, I don't like very much setting properties from a class from the code in another class. I think it is much more explicit to instantiate the class when you have all the data to instantiate, and then just pass the property value as a parameter to the instantiation. Is there any problem in doing it that way? 

(similar comment for `Index.es`, below.

Reply:
The idea here is to set the interval for all the child classes at once. All the metrics have a `query` object created using the `Query` class. 
This way we won't need to set the interval again and again in all the files for each of the metrics.